README.md

```diff
@@ -74,9 +76,9 @@ response = requests.get('http://en.wikipedia.org/', headers=headers, cookies=coo
 
 > I'd rather write programs to write programs than write programs.
 >
-> — Dick Sites, Digital Equipment Corporation, September 1985
+> — Dick Sites, Digital Equipment Corporation, 1985
```

Comment:
that's when [it was published](https://tildesites.bowdoin.edu/~ltoma/teaching/cs340/spring05/coursestuff/Bentley_BumperSticker.pdf), not when he said it